.kokoro/nightly/samples.cfg

```diff
@@ -34,5 +34,5 @@ env_vars: {
 
 env_vars: {
   key: "ENABLE_BUILD_COP"
-  value: "true"
```

Comment:
This will stop the flakybot report:

https://github.com/googleapis/java-bigqueryconnection/issues/832